src/applications/personalization/profile/components/notification-settings/NotificationSettings.jsx

```diff
@@ -82,14 +94,31 @@ const NotificationSettings = ({
           mobilePhoneNumber={mobilePhoneNumber}
         />
       ) : null}
+      {!shouldShowLoadingIndicator
+        ? notificationGroups.ids.map(groupId => {
+            if (groupId === 'group3') {
```

Comment:
I'm not crazy about filtering out the "your health care" notification group here and will likely revise it in my next PR that adds the edit functionality.